main.css

```diff
@@ -108,3 +121,2555 @@ body {
       font-size: 0.875rem; /* 14px */
       font-weight: 600;
     }
+
+.prog-bar{}
```

Comment:
**Firstly**
I'd prefer if this was called something like `prog-controls` or `programme-controls`
because I read `prog-bar` as "progress bar" which is something quite different and, though it is similar in form to the `.bar` block that already exists, I don't want anyone to draw an explicit association by name

Second:
Let's give this:
```css
border-top: solid 1px rgb(192, 192, 194);
border-bottom: solid 1px rgb(192, 192, 194);
```
to make a nice separation between controls and content